utils/keycodes.js

```diff
@@ -10,3 +20,115 @@ export const DOWN = 40;
 export const DELETE = 46;
 
 export const F10 = 121;
+
+export const ALT = 'alt';
+export const CTRL = 'ctrl';
+export const PRIMARY = 'mod';
+export const META = 'meta';
+export const SHIFT = 'shift';
+
+/**
+ * Return true if platform is MacOS.
+ *
+ * @param {Object} _window   window object by default; used for DI testing.
+ *
+ * @return {boolean}         True if MacOS; false otherwise.
+ */
+export function isMacOS( _window = window ) {
+	return _window.navigator.platform.indexOf( 'Mac' ) !== -1;
+}
+
+/**
+ * Create a keyboard shortcut based on a string of modifiers + key(s).
+ *
+ * This function is not intended to be used directly by developers.
+ * Instead, use primaryShortcut(), accessShortcut(), etc.
+ *
+ * @param {string} keys      Modifier and keyboard keys, seperated by "+".
+ * @param {Object} _isMacOS  isMacOS function by default; used for DI testing.
+ *
+ * @return {string}          The keyboard shortcut.
+ */
+export function keyboardShortcut( keys, _isMacOS = isMacOS ) {
+	const isMac = _isMacOS();
+
+	const alt = isMac ? '⌥' : 'Alt';
+	const meta = isMac ? '⌃' : '⊞';
```

Comment:
Just to note: I know this is a common way to display shortcuts on the Mac, but my built-in keyboard doesn't have any of these symbols, so this always confuses me. 🙈

Reply:
Huh, good point! Mine too; the only symbol is for Command, and this is a US keyboard:

![img_0831](https://user-images.githubusercontent.com/90871/39647308-4746cb14-4fd6-11e8-8a33-6a0cc669e1bf.jpg)

I just checked and it's not even standard on the Apple keyboards anymore. These symbols are cute and idiomatic for Mac users but they're not that helpful if you're new. Shall we just move them to strings instead?

Reply:
I don't mind keeping the command symbol since it's still on the keyboard (and I know that one), the rest I'm not so sure. I don't know how people make sense of the shortcuts shown on the Mac menus... :)

When I see `⌥` or `⌃` I just make a guess until I get what I want hahaha.

So... it's up to you. I just wanted to make a note here it may not make sense to some people.

Reply:
Not sure if someone will try to press the caret key. :D

Reply:
Yeah I was about to file an issue and ask people what they think, but I say we eschew Mac conventions here for usability. I'll keep the command glyph but change the rest to text. 👍